target/linux/ramips/dts/mt7620a_trendnet_tha103ac.dts

```diff
@@ -0,0 +1,199 @@
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "trendnet,tha103ac", "ralink,mt7620a-soc";
+	model = "TRENDNET THA-103AC (EU)";
+
+	aliases {
+		label-mac-device = &ethernet;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys: keys {
+		compatible = "gpio-keys";
+
+		button_reset {
+			label = "reset";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		/* wps button reassigned to rfkill */
+		button_rfkill {
+			label = "wifi on/off";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wps: led_0 {
```

Comment:
`led-0`, `led-1` ... are preferable.
```
patternProperties:
  "(^led-[0-9a-f]$|led)":
```
https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/leds-gpio.yaml

`led_wps` (and all other except `led_power`) labels are unused and can be dropped.

Reply:
All unused labels were removed and tested.